### ISSUE TITLE
Pretraining weights

### ISSUE THREAD

**Issue by sunshiding** [open-mmlab/mmsegmentation#1085]
Hello, when using pre-training weights, do I need to change "pretrained='open-mmlab://resnet50_v1c" to "pretrained='https://download.openmmlab.com/pretrain/third_party/resnet50_v1c-2cccc1ad.pth" to automatically load the weights?

If I do not make changes, the model weights will not be downloaded automatically?

**Comment by MengzhangLI** [comment]
Hi, thanks for your feedback.

Could you check your training log to see whether `init_cfg` or `pretrained` keys exist?

Maybe you can attach your training log here to help us check whether pretrained is loaded.

Thank you very much!

**Comment by cmFighting** [comment]
> 

Hello, I also encountered this problem. I found that the pre-trained model of the backbone network is not automatically loaded. This is the original training log. It can be found that although the parameters are updated to init_cfg, the pre-trained model is not actually loaded .

`
2021-12-01 10:25:10,157 - mmseg - INFO - Loaded 28000 images
/home/chenming/scm/code/st_seg/fffff/mmsegmentation-0.18.0/mmseg/models/backbones/resnet.py:432: UserWarning: DeprecationWarning: pretrained is a deprecated, please use "init_cfg" instead
  warnings.warn('DeprecationWarning: pretrained is a deprecated, '
/home/chenming/miniconda3/envs/k_net/lib/python3.7/site-packages/mmcv/utils/misc.py:324: UserWarning: "dropout" is deprecated in `FFN.__init__`, please use "ffn_drop" instead
  f'"{src_arg_name}" is deprecated in '
`

But when I add the following two lines at the end of resnet's __init__() function, the pre-trained model can be loaded correctly.

`
        logger = get_root_logger()
        load_checkpoint(self, pretrained, strict=False, logger=logger)
`

log is：
`
2021-12-01 10:25:51,036 - mmseg - INFO - Loaded 28000 images
/home/chenming/scm/code/st_seg/fffff/mmsegmentation-0.18.0/mmseg/models/backbones/resnet.py:432: UserWarning: DeprecationWarning: pretrained is a deprecated, please use "init_cfg" instead
  warnings.warn('DeprecationWarning: pretrained is a deprecated, '
2021-12-01 10:25:51,147 - mmseg - INFO - Use load_from_openmmlab loader
2021-12-01 10:25:51,537 - mmseg - WARNING - The model and loaded state dict do not match exactly

unexpected key in source state_dict: fc.weight, fc.bias

/home/chenming/miniconda3/envs/k_net/lib/python3.7/site-packages/mmcv/utils/misc.py:324: UserWarning: "dropout" is deprecated in `FFN.__init__`, please use "ffn_drop" instead
  f'"{src_arg_name}" is deprecated in '
`

**Comment by MengzhangLI** [comment]
> 

Hi, sorry to see that. Let us fix your problem ASAP.

I think it may be caused by our code upgrade from `pretrained` to `init_cfg`. 

(1) Which version of MMCV and MMSegmentation you use?

(2) Could you try to update MMCV and MMSegmentation to latest version (which may also need to be compatible for your server) to see whether this problem still exist?

Thanks!

**Comment by sunshiding** [comment]
> > 
> 
> Hi, sorry to see that. Let us fix your problem ASAP.
> 
> I think it may be caused by our code upgrade from `pretrained` to `init_cfg`.
> 
> (1) Which version of MMCV and MMSegmentation you use?
> 
> (2) Could you try to update MMCV and MMSegmentation to latest version (which may also need to be compatible for your server) to see whether this problem still exist?
> 
> Thanks!

Thank you very much for your patience, the problem has been solved perfectly.
